scripts/ci/pre_commit/pre_commit_breeze_cmd_line.py

```diff
@@ -79,6 +80,19 @@ def verify_all_commands_described_in_docs():
         sys.exit(1)
 
 
+def verify_breeze_installed():
+    result = subprocess.getoutput('breeze version')
+    if "Breeze installed from" in result:
+        print_help_for_all_commands()
+        verify_all_commands_described_in_docs()
+    else:
+        print(
+            "You don't have Breeze installed, follow BREEZE.rst to install it \
```

Comment:
we should use rich console and [red] for error and [yellow] for "action" to be done by the user